service-front/module/Application/view/layout/layout.phtml

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @var Laminas\View\Renderer\PhpRenderer $this
+ */
+?>
+<?= $this->doctype() ?>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <?= $this->headTitle('Laminas MVC Skeleton')->setSeparator(' - ')->setAutoEscape(false) ?>
+
+        <?= $this->headMeta()
+            ->appendName('viewport', 'width=device-width, initial-scale=1.0')
+            ->appendHttpEquiv('X-UA-Compatible', 'IE=edge')
+        ?>
+
+        <!-- Le styles -->
+        <?= $this->headLink([
+                                'rel' => 'shortcut icon',
+                                'type' => 'image/vnd.microsoft.icon',
+                                'href' => $this->basePath() . '/img/favicon.ico'
+                            ])
+            ->prependStylesheet($this->basePath('css/style.css'))
+            ->prependStylesheet($this->basePath('css/bootstrap.min.css'))
```

Comment:
I'd get rid of this CSS addition as well pretty quickly be moving to GDS styles. Same with any JS references.